### DISCUSSION TITLE
Display installiert und verkabelt - funktioniert aber nicht

### DISCUSSION THREAD

**Discussion by silvermaster5** [tbnobody/OpenDTU#845] [Q&A]
Hallo Community,

ich habe SSD1306 an den ESP32 angeschlossen, die  nodemcu_esp32.json eingespielt und das Profil für SSD1306 ausgewählt
Wie im Screenshoot zu sehen, werden auch keine Fehler in der Pinbelegung angezeigt. Das Display wird nicht aktiv.
![image](https://user-images.githubusercontent.com/131667082/234321094-4b912316-7e86-4337-a646-b576a0e58218.png)

Ich habe 
SCL auf GPIO22 
SCA auf GPIO21
GND auf GND 
VDO auf 5V

Auch ein Neustart hat nichts gebracht. Was habe ich übersehen? 

Danke
silvermaster

**Comment by DcFamas** [comment]
Gerne mal mit 3.3V testen.

**Reply by silvermaster5** [reply]
sorry, ich verstehe nicht was du meinst. Was soll ich tun?

**Reply by DcFamas** [reply]
Den Display einmal an den 3.3V Anschluss des ESP32 hängen und nicht an 5V. Sodass du ihn mit 3.3V betreibst und nicht mit 5V.

**Comment by DPO99** [comment]
> Hallo Community,
> 
> ich habe SSD1306 an den ESP32 angeschlossen, die  nodemcu_esp32.json eingespielt und das Profil für SSD1306 ausgewählt
> Wie im Screenshoot zu sehen, werden auch keine Fehler in der Pinbelegung angezeigt. Das Display wird nicht aktiv.
> ![image](https://user-images.githubusercontent.com/131667082/234321094-4b912316-7e86-4337-a646-b576a0e58218.png)
> 
> Ich habe 
> SCL auf GPIO22 
> SCA auf GPIO21
> GND auf GND 
> VDO auf 5V
> 
> Auch ein Neustart hat nichts gebracht. Was habe ich übersehen? 
> 
> Danke
> silvermaster

Der Screenshot zeigt m.e. niemals Fehler nur die gespeicherte Konfiguration. Von daher ggf nochmal Pin Belegung des Bildschirms und Verkabelung zum esp32 prüfen.
